app/src/main/java/com/github/se/eduverse/ui/navigation/NavigationActions.kt

```diff
@@ -0,0 +1,105 @@
+// This file was copied and adapted from the bootcamp solution
+
+package com.github.se.eduverse.ui.navigation
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.CameraAlt
+import androidx.compose.material.icons.outlined.Home
+import androidx.compose.material.icons.outlined.Menu
+import androidx.compose.material.icons.outlined.PlayArrow
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.navigation.NavGraph.Companion.findStartDestination
+import androidx.navigation.NavHostController
+
+object Route {
+  const val AUTH = "Auth"
+  const val DASHBOARD = "Dashboard"
+  const val VIDEOS = "Videos"
+  const val CAMERA = "Camera"
+  const val OTHERS = "Others"
+}
+
+object Screen {
+  const val AUTH = "Auth Screen"
+  const val DASHBOARD = "Dashboard screen"
+  const val VIDEOS = "Videos screen"
+  const val CAMERA = "Camera screen"
+  const val OTHERS = "Others screen"
+  const val EDIT_PROFILE = "EditProfile screen"
+  const val FOLDER = "Folder screen"
+  const val COURSES = "Courses screen"
+}
+
+data class TopLevelDestination(val route: String, val icon: ImageVector, val textId: String)
+
+object TopLevelDestinations {
+  val DASHBOARD =
+      TopLevelDestination(route = Route.DASHBOARD, icon = Icons.Outlined.Home, textId = "Dashboard")
+  val VIDEOS =
+      TopLevelDestination(route = Route.VIDEOS, icon = Icons.Outlined.PlayArrow, textId = "Videos")
+  val CAMERA =
+      TopLevelDestination(route = Route.CAMERA, icon = Icons.Outlined.CameraAlt, textId = "Camera")
```

Comment:
I don't find any CameraAlt in Icons.Outlined, did you do something special to get it ?